common/dna_data/wc_orc_dna.txt

```diff
@@ -1893,3 +1893,124 @@ griselda_dna = { # by icelille999
 	}
 	enabled=yes
 }
+
+garrosh_hellscream_dna = {
```

Comment:
You create some visible clipping here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123140596-864fdc00-d468-11eb-86dc-57822bc17be2.png)

</details>

Reply:
I would work a bit more on Garrosh.
Change his eyes to red (orange).
Work on his cheeks.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123141018-f9f1e900-d468-11eb-8001-d0f89f5d5c0d.png)

</details>